### ISSUE TITLE
If the warning message for an input is too far right, it gets squashed 

### ISSUE THREAD

**Issue by christianp** [numbas/Numbas#457]
A student had a very long answer to a JME question, which pushed the warning tool tip to the right-hand edge of the window. The tool tip should be positioned so that it never gets too narrow. 
It's probably worth finding a decent library to do the tooltips rather than fixing my hacked-together code.
